pkg/daemon/ceph/client/filesystem.go

```diff
@@ -191,6 +191,37 @@ func SetNumMDSRanks(context *clusterd.Context, clusterInfo *ClusterInfo, fsName
 	return nil
 }
 
+// FailAllStandbyReplayMDS: fail all mds in up:standby-replay state
+func FailAllStandbyReplayMDS(context *clusterd.Context, clusterInfo *ClusterInfo, fsName string) error {
+	fs, err := GetFilesystem(context, clusterInfo, fsName)
+	if err != nil {
+		return errors.Wrapf(err, "failed to get ceph fs dump")
+	}
+	for _, info := range fs.MDSMap.Info {
+		if info.State == "up:standby-replay" {
+			if err := FailMDS(context, clusterInfo, info.GID); err != nil {
+				return errors.Wrapf(err, "failed to fail mds in up:standby-replay state")
```

Comment:
Is there more identifying information from `info` that can be added here to help identify which MDS is problematic?

Reply:
looks like there are other [states](https://github.com/ceph/ceph/blob/7bc9206457ae616dd6aa9508a4cb38e8f03b34ca/src/include/ceph_fs.h#L292) and [names](https://github.com/ceph/ceph/blob/7bc9206457ae616dd6aa9508a4cb38e8f03b34ca/src/common/ceph_strings.cc#L227) available, but not quite sure how to handle them... any idea?

Reply:
```suggestion
				return errors.Wrapf(err, "failed to fail MDS %q for filesystem %q in up:standby-replay state", info.Name, fsName)
```